### PR TITLE
#74 header component

### PR DESCRIPTION
**Pull request by LukaszAmbrozewski**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -112,6 +113,10 @@ export const App = () => {
           path="/student"
           element={<StudentView />}
         />
+        <Route
+          path="/hr/profile"
```
Comment (mae1579) [inline src/App.tsx:117]: Great packages.

File: src/common/MenuBtn/MenuBtn.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    margin-left: 0;
+    padding: 0 0 0 15px;
+    text-decoration: none;
+    text-align: left;
+    cursor: pointer;
+}
+
+.menu-btn:hover {
+    background: #e02735;
+}
```
Comment (mae1579) [inline src/common/MenuBtn/MenuBtn.css:18]: great css.

File: src/common/MenuBtn/MenuBtn.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      onClick={props.onClick}>
+      {props.text}
+    </Link>
+  ) : (
+    <button
+      className="menu-btn"
+      onClick={props.onClick}>
+      {props.text}
+    </button>
+  );
```
Comment (mae1579) [inline src/common/MenuBtn/MenuBtn.tsx:26]: keep doing that :)

File: src/common/ShowMenuBtn/ShowMenuBtn.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+export const ShowMenuBtn = (props: Props) => {
+  return (
+    <button
+      className="show-menu-btn"
+      onClick={props.onClick}>
+      <ArrowDropDownIcon />
+    </button>
+  );
+};
```
Comment (mae1579) [inline src/common/ShowMenuBtn/ShowMenuBtn.tsx:18]: very nice man 👍

File: src/common/UserPhoto/UserPhoto.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className={props.className}>
+      <img
+        src="https://images.unsplash.com/photo-1570295999919-56ceb5ecca61?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80"
+        alt="Zdjęcie użytkownika"
+        className="photo"
+      />
+    </div>
+  );
+};
```
Comment (mae1579) [inline src/common/UserPhoto/UserPhoto.tsx:19]: we are gonna check leter that , i mean  photo of avatar  :) great job

File: src/components/Header/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+          <UserPhoto className="photo-space" />
+          <div className="user-name">
+            <p>Mateusz Kowalski</p>
+          </div>
+          <HeaderMenu className="options-btn" />
+        </div>
+      </div>
+    </>
+  );
+};
```
Comment (mae1579) [inline src/components/Header/Header.tsx:25]: Great job

File: src/components/HeaderMenu/HeaderMenu.tsx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+        <MenuBtn
+          text="Konto"
+          to="/hr/profile"></MenuBtn>
+        <MenuBtn
+          text="Wyloguj"
+          onClick={handleLogout}></MenuBtn>
+      </StyledMenu>
+    </div>
+  );
+};
```
Comment (mae1579) [inline src/components/HeaderMenu/HeaderMenu.tsx:94]: great component :)

**Review by mae1579 — COMMENTED** [review]
all code is great :), keep doing that man 🥇

File: src/common/UserPhoto/UserPhoto.tsx (hunk 5 above)
Comment (patrycja-dev41171) [inline src/common/UserPhoto/UserPhoto.tsx:19]: ok, nice code.

File: src/components/Header/Header.css [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.user-name {
+    color: white;
+}
+
+.options-btn {
+    padding-top: 3px;
+}
+
```
Comment (patrycja-dev41171) [inline src/components/Header/Header.css:39]: Using css classes, good!

File: src/components/Header/Header.tsx (hunk 6 above)
Comment (patrycja-dev41171) [inline src/components/Header/Header.tsx:25]: Good.

**Review by patrycja-dev41171 — COMMENTED** [review]
Good job 👍

**Pull request merged**